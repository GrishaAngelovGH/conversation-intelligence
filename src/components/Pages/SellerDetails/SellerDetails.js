const SellerDetails = () => (
  <div className="row">
    <div className="col-md-12">
      SellerDetails
    </div>
  </div>
)

export default SellerDetails