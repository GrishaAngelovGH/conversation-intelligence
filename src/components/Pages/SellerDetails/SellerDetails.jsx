import Table from 'react-bootstrap/Table'

import Attendee from 'components/Pages/common/Attendee'

export const StatisticSummary = () => (
  <div className='row mt-3 border rounded p-3 bg-light justify-content-center text-center'>
    <div className='col-md-2'>
      <div className='fw-bold text-muted'>Talk to listen ratio</div>
      <div className='fw-bold'>52/48%</div>
    </div>

    <div className='col-md-2'>
      <div className='fw-bold text-muted'>Talking speed</div>
      <div className='fw-bold'>156.8 wpm</div>
    </div>

    <div className='col-md-2'>
      <div className='fw-bold text-muted'>Switches per hour</div>
      <div className='fw-bold'>32.72</div>
    </div>

    <div className='col-md-3'>
      <div className='fw-bold text-muted'>Pause before speaking</div>
      <div className='fw-bold'>68.97ms</div>
    </div>

    <div className='col-md-3'>
      <div className='fw-bold text-muted'>Longest customer monologue</div>
      <div className='fw-bold'>37sec</div>
    </div>
  </div>
)

export const SellerInsights = () => (
  <div className='row mt-3'>
    <div className='col-md-12'>
      <h4>Seller Insights</h4>

      <div className='row gap-2 text-center'>
        <div className='col border border-primary rounded p-2'>
          James Smith has been talking too fast in most conversations
        </div>

        <div className='col border border-primary rounded p-2'>
          James Smith&apos;s talk-to-listen ratio is 13% higher than the team&apos;s average
        </div>

        <div className='col border border-primary rounded p-2'>
          Positive customer sentiments is higher than the average in 38% of calls
        </div>
      </div>

    </div>
  </div>
)

export const CallHistory = () => {
  const data = [
    { id: 1, title: 'Closing the deal', added: '6 hours ago', duration: '31m 30s', phone: '123456789', relatedOpportunity: 'Not Set' },
    { id: 2, title: 'Initial discussion', added: '3 days ago', duration: '35m 14s', phone: '123456789', relatedOpportunity: 'Not Set' },
    { id: 3, title: 'Initial catalog launch', added: '3 days ago', duration: '45m 20s', phone: '123456789', relatedOpportunity: 'Not Set' },
  ]

  return (
    <div className='row mt-3'>
      <div className='col-md-12'>
        <h4>Call History</h4>

        <Table bordered hover>
          <thead>
            <tr>
              <th></th>
              <th>Title</th>
              <th>Added</th>
              <th>Duration</th>
              <th>Phone Number</th>
              <th>Related Opportunity</th>
            </tr>
          </thead>
          <tbody>
            {
              data.map(v => (
                <tr key={v.id}>
                  <td><i className='bi bi-telephone'></i></td>
                  <td>{v.title}</td>
                  <td>{v.added}</td>
                  <td>{v.duration}</td>
                  <td>{v.phone}</td>
                  <td>{v.relatedOpportunity}</td>
                </tr>
              ))
            }
          </tbody>
        </Table>
      </div>
    </div>
  )
}

const SellerDetails = () => (
  <div className='row m-3'>
    <div className='col-md-12'>
      <h3 className='fw-bold'>Seller Details</h3>

      <div className='row'>
        <div className='col-md-5'>
          <Attendee shortName='JS' fullName='James Smith' />
        </div>
      </div>

      <StatisticSummary />

      <SellerInsights />

      <CallHistory />
    </div>
  </div>
)

export default SellerDetails