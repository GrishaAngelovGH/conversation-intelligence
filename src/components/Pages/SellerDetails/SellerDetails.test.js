import { render } from '@testing-library/react'

import SellerDetails, { StatisticSummary, SellerInsights, CallHistory } from './SellerDetails'

test('should render component', () => {
  const container = render(<SellerDetails />)

  expect(container).toMatchSnapshot()
})

test('should render StatisticSummary', () => {
  const container = render(<StatisticSummary />)

  expect(container).toMatchSnapshot()
})

test('should render SellerInsights', () => {
  const container = render(<SellerInsights />)

  expect(container).toMatchSnapshot()
})

test('should render CallHistory', () => {
  const container = render(<CallHistory />)

  expect(container).toMatchSnapshot()
})