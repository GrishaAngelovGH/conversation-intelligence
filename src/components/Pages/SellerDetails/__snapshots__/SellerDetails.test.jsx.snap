// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render CallHistory component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row mt-3"
      >
        <div
          class="col-md-12"
        >
          <h4>
            Call History
          </h4>
          <table
            class="table table-bordered table-hover"
          >
            <thead>
              <tr>
                <th />
                <th>
                  Title
                </th>
                <th>
                  Added
                </th>
                <th>
                  Duration
                </th>
                <th>
                  Phone Number
                </th>
                <th>
                  Related Opportunity
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  <i
                    class="bi bi-telephone"
                  />
                </td>
                <td>
                  Closing the deal
                </td>
                <td>
                  6 hours ago
                </td>
                <td>
                  31m 30s
                </td>
                <td>
                  123456789
                </td>
                <td>
                  Not Set
                </td>
              </tr>
              <tr>
                <td>
                  <i
                    class="bi bi-telephone"
                  />
                </td>
                <td>
                  Initial discussion
                </td>
                <td>
                  3 days ago
                </td>
                <td>
                  35m 14s
                </td>
                <td>
                  123456789
                </td>
                <td>
                  Not Set
                </td>
              </tr>
              <tr>
                <td>
                  <i
                    class="bi bi-telephone"
                  />
                </td>
                <td>
                  Initial catalog launch
                </td>
                <td>
                  3 days ago
                </td>
                <td>
                  45m 20s
                </td>
                <td>
                  123456789
                </td>
                <td>
                  Not Set
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row mt-3"
    >
      <div
        class="col-md-12"
      >
        <h4>
          Call History
        </h4>
        <table
          class="table table-bordered table-hover"
        >
          <thead>
            <tr>
              <th />
              <th>
                Title
              </th>
              <th>
                Added
              </th>
              <th>
                Duration
              </th>
              <th>
                Phone Number
              </th>
              <th>
                Related Opportunity
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                <i
                  class="bi bi-telephone"
                />
              </td>
              <td>
                Closing the deal
              </td>
              <td>
                6 hours ago
              </td>
              <td>
                31m 30s
              </td>
              <td>
                123456789
              </td>
              <td>
                Not Set
              </td>
            </tr>
            <tr>
              <td>
                <i
                  class="bi bi-telephone"
                />
              </td>
              <td>
                Initial discussion
              </td>
              <td>
                3 days ago
              </td>
              <td>
                35m 14s
              </td>
              <td>
                123456789
              </td>
              <td>
                Not Set
              </td>
            </tr>
            <tr>
              <td>
                <i
                  class="bi bi-telephone"
                />
              </td>
              <td>
                Initial catalog launch
              </td>
              <td>
                3 days ago
              </td>
              <td>
                45m 20s
              </td>
              <td>
                123456789
              </td>
              <td>
                Not Set
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`should render SellerDetails component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row m-3"
      >
        <div
          class="col-md-12"
        >
          <h3
            class="fw-bold"
          >
            Seller Details
          </h3>
          <div
            class="row"
          >
            <div
              class="col-md-5"
            >
              <div
                class="row mt-1 align-items-center"
              >
                <div
                  class="col-md-2"
                >
                  <div
                    class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                    style="width: 40px; height: 40px;"
                  >
                    JS
                  </div>
                </div>
                <div
                  class="col-md-6"
                >
                  <h4
                    class="m-0"
                  >
                    James Smith
                  </h4>
                </div>
              </div>
            </div>
          </div>
          <div
            class="row mt-3 border rounded p-3 bg-light justify-content-center text-center"
          >
            <div
              class="col-md-2"
            >
              <div
                class="fw-bold text-muted"
              >
                Talk to listen ratio
              </div>
              <div
                class="fw-bold"
              >
                52/48%
              </div>
            </div>
            <div
              class="col-md-2"
            >
              <div
                class="fw-bold text-muted"
              >
                Talking speed
              </div>
              <div
                class="fw-bold"
              >
                156.8 wpm
              </div>
            </div>
            <div
              class="col-md-2"
            >
              <div
                class="fw-bold text-muted"
              >
                Switches per hour
              </div>
              <div
                class="fw-bold"
              >
                32.72
              </div>
            </div>
            <div
              class="col-md-3"
            >
              <div
                class="fw-bold text-muted"
              >
                Pause before speaking
              </div>
              <div
                class="fw-bold"
              >
                68.97ms
              </div>
            </div>
            <div
              class="col-md-3"
            >
              <div
                class="fw-bold text-muted"
              >
                Longest customer monologue
              </div>
              <div
                class="fw-bold"
              >
                37sec
              </div>
            </div>
          </div>
          <div
            class="row mt-3"
          >
            <div
              class="col-md-12"
            >
              <h4>
                Seller Insights
              </h4>
              <div
                class="row gap-2 text-center"
              >
                <div
                  class="col border border-primary rounded p-2"
                >
                  James Smith has been talking too fast in most conversations
                </div>
                <div
                  class="col border border-primary rounded p-2"
                >
                  James Smith's talk-to-listen ratio is 13% higher than the team's average
                </div>
                <div
                  class="col border border-primary rounded p-2"
                >
                  Positive customer sentiments is higher than the average in 38% of calls
                </div>
              </div>
            </div>
          </div>
          <div
            class="row mt-3"
          >
            <div
              class="col-md-12"
            >
              <h4>
                Call History
              </h4>
              <table
                class="table table-bordered table-hover"
              >
                <thead>
                  <tr>
                    <th />
                    <th>
                      Title
                    </th>
                    <th>
                      Added
                    </th>
                    <th>
                      Duration
                    </th>
                    <th>
                      Phone Number
                    </th>
                    <th>
                      Related Opportunity
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>
                      <i
                        class="bi bi-telephone"
                      />
                    </td>
                    <td>
                      Closing the deal
                    </td>
                    <td>
                      6 hours ago
                    </td>
                    <td>
                      31m 30s
                    </td>
                    <td>
                      123456789
                    </td>
                    <td>
                      Not Set
                    </td>
                  </tr>
                  <tr>
                    <td>
                      <i
                        class="bi bi-telephone"
                      />
                    </td>
                    <td>
                      Initial discussion
                    </td>
                    <td>
                      3 days ago
                    </td>
                    <td>
                      35m 14s
                    </td>
                    <td>
                      123456789
                    </td>
                    <td>
                      Not Set
                    </td>
                  </tr>
                  <tr>
                    <td>
                      <i
                        class="bi bi-telephone"
                      />
                    </td>
                    <td>
                      Initial catalog launch
                    </td>
                    <td>
                      3 days ago
                    </td>
                    <td>
                      45m 20s
                    </td>
                    <td>
                      123456789
                    </td>
                    <td>
                      Not Set
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row m-3"
    >
      <div
        class="col-md-12"
      >
        <h3
          class="fw-bold"
        >
          Seller Details
        </h3>
        <div
          class="row"
        >
          <div
            class="col-md-5"
          >
            <div
              class="row mt-1 align-items-center"
            >
              <div
                class="col-md-2"
              >
                <div
                  class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                  style="width: 40px; height: 40px;"
                >
                  JS
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <h4
                  class="m-0"
                >
                  James Smith
                </h4>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mt-3 border rounded p-3 bg-light justify-content-center text-center"
        >
          <div
            class="col-md-2"
          >
            <div
              class="fw-bold text-muted"
            >
              Talk to listen ratio
            </div>
            <div
              class="fw-bold"
            >
              52/48%
            </div>
          </div>
          <div
            class="col-md-2"
          >
            <div
              class="fw-bold text-muted"
            >
              Talking speed
            </div>
            <div
              class="fw-bold"
            >
              156.8 wpm
            </div>
          </div>
          <div
            class="col-md-2"
          >
            <div
              class="fw-bold text-muted"
            >
              Switches per hour
            </div>
            <div
              class="fw-bold"
            >
              32.72
            </div>
          </div>
          <div
            class="col-md-3"
          >
            <div
              class="fw-bold text-muted"
            >
              Pause before speaking
            </div>
            <div
              class="fw-bold"
            >
              68.97ms
            </div>
          </div>
          <div
            class="col-md-3"
          >
            <div
              class="fw-bold text-muted"
            >
              Longest customer monologue
            </div>
            <div
              class="fw-bold"
            >
              37sec
            </div>
          </div>
        </div>
        <div
          class="row mt-3"
        >
          <div
            class="col-md-12"
          >
            <h4>
              Seller Insights
            </h4>
            <div
              class="row gap-2 text-center"
            >
              <div
                class="col border border-primary rounded p-2"
              >
                James Smith has been talking too fast in most conversations
              </div>
              <div
                class="col border border-primary rounded p-2"
              >
                James Smith's talk-to-listen ratio is 13% higher than the team's average
              </div>
              <div
                class="col border border-primary rounded p-2"
              >
                Positive customer sentiments is higher than the average in 38% of calls
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mt-3"
        >
          <div
            class="col-md-12"
          >
            <h4>
              Call History
            </h4>
            <table
              class="table table-bordered table-hover"
            >
              <thead>
                <tr>
                  <th />
                  <th>
                    Title
                  </th>
                  <th>
                    Added
                  </th>
                  <th>
                    Duration
                  </th>
                  <th>
                    Phone Number
                  </th>
                  <th>
                    Related Opportunity
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    <i
                      class="bi bi-telephone"
                    />
                  </td>
                  <td>
                    Closing the deal
                  </td>
                  <td>
                    6 hours ago
                  </td>
                  <td>
                    31m 30s
                  </td>
                  <td>
                    123456789
                  </td>
                  <td>
                    Not Set
                  </td>
                </tr>
                <tr>
                  <td>
                    <i
                      class="bi bi-telephone"
                    />
                  </td>
                  <td>
                    Initial discussion
                  </td>
                  <td>
                    3 days ago
                  </td>
                  <td>
                    35m 14s
                  </td>
                  <td>
                    123456789
                  </td>
                  <td>
                    Not Set
                  </td>
                </tr>
                <tr>
                  <td>
                    <i
                      class="bi bi-telephone"
                    />
                  </td>
                  <td>
                    Initial catalog launch
                  </td>
                  <td>
                    3 days ago
                  </td>
                  <td>
                    45m 20s
                  </td>
                  <td>
                    123456789
                  </td>
                  <td>
                    Not Set
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`should render SellerInsights component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row mt-3"
      >
        <div
          class="col-md-12"
        >
          <h4>
            Seller Insights
          </h4>
          <div
            class="row gap-2 text-center"
          >
            <div
              class="col border border-primary rounded p-2"
            >
              James Smith has been talking too fast in most conversations
            </div>
            <div
              class="col border border-primary rounded p-2"
            >
              James Smith's talk-to-listen ratio is 13% higher than the team's average
            </div>
            <div
              class="col border border-primary rounded p-2"
            >
              Positive customer sentiments is higher than the average in 38% of calls
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row mt-3"
    >
      <div
        class="col-md-12"
      >
        <h4>
          Seller Insights
        </h4>
        <div
          class="row gap-2 text-center"
        >
          <div
            class="col border border-primary rounded p-2"
          >
            James Smith has been talking too fast in most conversations
          </div>
          <div
            class="col border border-primary rounded p-2"
          >
            James Smith's talk-to-listen ratio is 13% higher than the team's average
          </div>
          <div
            class="col border border-primary rounded p-2"
          >
            Positive customer sentiments is higher than the average in 38% of calls
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`should render StatisticSummary component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row mt-3 border rounded p-3 bg-light justify-content-center text-center"
      >
        <div
          class="col-md-2"
        >
          <div
            class="fw-bold text-muted"
          >
            Talk to listen ratio
          </div>
          <div
            class="fw-bold"
          >
            52/48%
          </div>
        </div>
        <div
          class="col-md-2"
        >
          <div
            class="fw-bold text-muted"
          >
            Talking speed
          </div>
          <div
            class="fw-bold"
          >
            156.8 wpm
          </div>
        </div>
        <div
          class="col-md-2"
        >
          <div
            class="fw-bold text-muted"
          >
            Switches per hour
          </div>
          <div
            class="fw-bold"
          >
            32.72
          </div>
        </div>
        <div
          class="col-md-3"
        >
          <div
            class="fw-bold text-muted"
          >
            Pause before speaking
          </div>
          <div
            class="fw-bold"
          >
            68.97ms
          </div>
        </div>
        <div
          class="col-md-3"
        >
          <div
            class="fw-bold text-muted"
          >
            Longest customer monologue
          </div>
          <div
            class="fw-bold"
          >
            37sec
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row mt-3 border rounded p-3 bg-light justify-content-center text-center"
    >
      <div
        class="col-md-2"
      >
        <div
          class="fw-bold text-muted"
        >
          Talk to listen ratio
        </div>
        <div
          class="fw-bold"
        >
          52/48%
        </div>
      </div>
      <div
        class="col-md-2"
      >
        <div
          class="fw-bold text-muted"
        >
          Talking speed
        </div>
        <div
          class="fw-bold"
        >
          156.8 wpm
        </div>
      </div>
      <div
        class="col-md-2"
      >
        <div
          class="fw-bold text-muted"
        >
          Switches per hour
        </div>
        <div
          class="fw-bold"
        >
          32.72
        </div>
      </div>
      <div
        class="col-md-3"
      >
        <div
          class="fw-bold text-muted"
        >
          Pause before speaking
        </div>
        <div
          class="fw-bold"
        >
          68.97ms
        </div>
      </div>
      <div
        class="col-md-3"
      >
        <div
          class="fw-bold text-muted"
        >
          Longest customer monologue
        </div>
        <div
          class="fw-bold"
        >
          37sec
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
